api/db/api.db.changelog.yml

```diff
@@ -30,4 +30,15 @@ databaseChangeLog:
                   constraints:
                     nullable: false
             tableName: policy_json
+   - changeSet:
+      id: 2
+      author: fujitsu
+      changes:
+      - update:
+          columns:
+          - column:
+              name: guid
+              type: varchar(50)
+              value: "1"
```

Comment:
Just a question: Wouldn't this update the guid column for all rows to 1 and is that ok?